### PR TITLE
Create a REPL for test/debugging

### PR DESCRIPTION
**Pull request by cpcallen**
At last: command-line javascript(-in-javascript)!

### CODE REVIEW

File: server/repl [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env node
+/**
+ * @license
+ * Code City: JavaScript Read Eval Print Loop
+ *
+ * Copyright 2013 Google Inc.
```
Comment (NeilFraser) [inline server/repl]: Practicing being a Time Lord again?

File: server/repl [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+});
+
+rl.prompt();
+rl.on('line', function(line) {
+  intrp.value = undefined;
+  intrp.createThread(line);
+  intrp.run();
+  console.log(intrp.pseudoToNative(intrp.value));
+  rl.prompt();
+}).on('close', () => {
```
Comment (NeilFraser) [inline server/repl]: Until we start supporting ES6 in Code City, let's not use ES6 where ES5 is just as easy to use.

File: server/repl (hunk 1 above)
Comment (cpcallen) [reply server/repl]: Fixed.

File: server/repl (hunk 2 above)
Comment (cpcallen) [reply server/repl]: Yeah.  Code was copied for documentation for `readline`; I fixed the one above but missed this one.

**Pull request merged**